src/shared/components/palette/palette.module.ts

```diff
@@ -16,7 +16,7 @@ import { ComplexModule } from './complex/complex.module';
 import { AddressModule } from './address/address.module';
 import { BaseFieldModule } from './base-field/base-field.module';
 import { WriteTextFieldComponent } from './text/write-text-field.component';
-import { ReactiveFormsModule } from '@angular/forms';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
```

Comment:
_Nitpick:_
Doesn't look like `FormsModule` is being used anywhere, which would make this import redundant.